{"name": "test_failGetElement", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <test_KWA_demo.test_KWA_Demo testMethod=test_failGetElement>\n\n    @qase.title('testFailGetElement')\n    def test_failGetElement(self):\n>       self.commands.testFailGetElement()\n\ntest_KWA_demo.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <KWA_demo_commands.DemoCommands object at 0x000001B04C935B40>\n\n    def testFailGetElement(self):\n>       assert self.getElement(self._AutoSuggest) != None\nE       AssertionError\n\nKWA_demo_commands.py:86: AssertionError"}, "attachments": [{"source": "c099d4fc-c36f-4f91-acba-5696a7f796fa-attachment.png", "type": "image/png"}, {"name": "log", "source": "c1d763c3-0626-4368-9a8a-1e3cce496f30-attachment.txt", "type": "text/plain"}], "start": 1667219185884, "stop": 1667219211422, "uuid": "071d6c3a-2679-4ae4-b2a8-f502b8386757", "historyId": "8da06a45c3678cd1e762e9eceade2cb0", "testCaseId": "6389d253f16a39b2d0e5481eaa902445", "fullName": "test_KWA_demo.test_KWA_Demo#test_failGetElement", "labels": [{"name": "tag", "value": "qase_title(title='testFailGetElement')"}, {"name": "tag", "value": "@pytest.mark.usefixtures('createClassDriver')"}, {"name": "suite", "value": "test_KWA_demo"}, {"name": "subSuite", "value": "test_KWA_Demo"}, {"name": "host", "value": "acer-laptop"}, {"name": "thread", "value": "12624-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_KWA_demo"}]}